### DISCUSSION TITLE
最新版本青龙报错

### DISCUSSION THREAD

**Discussion by Qscp** [RayWangQvQ/BiliBiliToolPro#767] [Q&A]
/ql/shell/share.sh: line 464: task_after: unbound variable

**Comment by STZMYSH** [comment]
+1,同样的问题

**Comment by ascte** [comment]
我更新青龙至2.17.12后没有该报错了，但是推送一直有问题，用的企业微信机器人，推送时查看日志报错如下：
========================

 v2.1.2 开源 by https://github.com/RayWangQvQ/BiliBiliToolPro

[16:37:26 INF] ·开始推送·VipBigPoint任务·
2024-11-29T08:37:26.7088643Z **Exception while emitting periodic batch from Ray.Serilog.Sinks.WorkWeiXinBatched.WorkWeiXinBatchedSink: Invalid URI: The format of the URI could not be determined.**
[16:37:26 INF] 运行环境：Production
[16:37:26 INF] 应用目录：/ql/data/repo/RayWangQvQ_BiliBiliToolPro/src/Ray.BiliBiliTool.Console/bin/Debug/net6.0/

[16:37:26 INF] 运行结束

## 执行结束... 2024-11-29 16:37:26  耗时 29 秒

**Comment by KosLzz** [comment]
最新2.18.3青龙,每天自动删除任务?????????????


## 开始执行... 2025-03-29 01:42:58
开始拉取仓库 RayWangQvQ_BiliBiliToolPro 到 /ql/data/repo/RayWangQvQ_BiliBiliToolPro
拉取 RayWangQvQ_BiliBiliToolPro 成功...
检测到有失效的定时任务:
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_base.sh
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_daily.sh
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_liveFansMedal.sh
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_liveLottery.sh
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_login.sh
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_test.sh
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_tryFix.sh
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_unfollowBatched.sh
RayWangQvQ_BiliBiliToolPro/qinglong/DefaultTasks/bili_task_vipBigPoint.sh
开始尝试自动删除失效的定时任务...
